VPN UPDATE (#27)
sheharyaar/iit-kgp-network: README_technical.md

|VPN|Platform|Status| Reason |
|---|---|---| --- |

> | [HotspotShield](https://www.hotspotshield.com/) | ![w] ![l] ![m] | ✅ | It's paid but the _free version_ is pretty good enough for daily usage. Can stream 4k videos with lag of about half a second ( if you test it by hopping to different timelines ). 2mbps limit on free version - _they say_ - **is actually about 85-100 mbps** (actual speed; whilist [ookla results showed 300mbps+ on an average at JCB Hall](https://www.speedtest.net/result/13836481406). It's `hydra` protocol connects quickly while `IKEv2 (IPSec)` doesn't work. Though being quite fast, pings are too high thus making high-end gaming next to impossible. Along with that, mobile version of it has 15 minutes timeout and have to see an ad to extend it furher by 15 minutes but there is no such case with desktop version. Hence, **it's a recommended choice for Desktop but NOT for mobile devices**. |

Reviewer: ```suggestion
| [HotspotShield](https://www.hotspotshield.com/) | ![w] ![l] ![m] | ✅ | It's paid but the _free version_ is pretty good enough for daily usage. Can stream 4k videos with a lag of about half a second ( if you test it by hopping to different timelines ). 2mbps limit on the free version - _they say_ - **is actually about 85-100 Mbps** (actual speed; whilst [ookla results showed 300mbps+ on an average at JCB Hall](https://www.speedtest.net/result/13836481406). Its `hydra` protocol connects quickly while `IKEv2 (IPSec)` doesn't work. Though being quite fast, pings are too high thus making high-end gaming next to impossible. Along with that, the mobile version of it has 15 minutes timeout and has to see an ad to extend it further by 15 minutes but there is no such case with the desktop version. Hence, **it's a recommended choice for Desktop but NOT for mobile devices**. |
```

| [NoCardVPN](https://play.google.com/store/search?q=no%20cardvpn&c=apps&hl=en_IN&gl=US) |  ![a] | ✅ | Works like a charm with no significant loss in speed, but have to manually increase connection time (10/20/30/40/60 mins depending on your luck; else it will disconnect after that time is over) during which they serve ads for the survival of the project since it is completely free on user's end. |

ℹ️ For detailed comparison of OpenVPN on EC2, DigitalOcean, ExpressVPN and Speedify for `gamers` and `casual users`, see [OpenVPN vs ExpressVPN vs Speedify](#54-OpenVPN-vs-ExpressVPN-vs-Speedify) section. 

- Working Solutions but **not recommended**:

|VPN|Platform|Status| Reason |
|---|---|---| --- |
| [Psiphon](https://play.google.com/store/search?q=psiphon&c=apps&hl=en_IN&gl=US) | ![a] | ✅ | Uses `L2TP/IPsec`. For more info on protocols see [VPN Protocols](#42-vpn-protocols) section.<br/>Slow and requires more CPU consumption. |

> | [Tor via Brave Browser](https://support.brave.com/hc/en-us/articles/360018121491-What-is-a-Private-Window-with-Tor-Connectivity-) | ![w] ![l] ![m] | ✅ | It just works - Very slow ( less than 1 mbps; tested on ookla ) - Can just somehow stream 720p detailed motion videos - Captcha issues while Surfing the web; since TOR network so, obviously - Cloudflare bots protection checking leads to more delay in opening websites. But it works! |

Reviewer: ```suggestion
| [Tor via Brave Browser](https://support.brave.com/hc/en-us/articles/360018121491-What-is-a-Private-Window-with-Tor-Connectivity-) | ![w] ![l] ![m] | ✅ | It just works - Very slow ( less than 1 Mbps; tested on Ookla ) - Can just somehow stream 720p detailed motion videos - Captcha issues while Surfing the web; since TOR network so, obviously - Cloudflare bots protection checking leads to more delay in opening websites. But it works! |
```

| [ExpressVPN](https://www.expressvpn.com/) (Not free) | ![w] ![l] ![a] ![m] | ✅ | Even though paid, it's fast, stable and the secure option out in the internet but with [Update in Indian laws to store logs for servers in India](https://www.techradar.com/news/any-vpn-with-servers-in-india-must-now-store-activity-logs-on-users), ExpressVPN has removed its servers and the latency has been increased upto 120ms thus making it unsuitable for gaming. |
| [HotspotShield](https://www.hotspotshield.com/) | ![w] ![l] ![a] ![m] | ✅ | It's paid but the _free version_ is pretty good enough for daily usage. Can stream 4k videos with lag of about half a second ( if you test it by hopping to different timelines ). 2mbps limit on free version - they say - is actually about 15-17 mbps (actual speed; whilist [ookla showed 55mbps on an average](https://www.speedtest.net/result/13836205312) ) mbps when tested at JCB Hall. It's `hydra` protocol connects quickly while `IKEv2 (IPSec)` doesn't work. |

> | [Tor via Brave Browser](https://support.brave.com/hc/en-us/articles/360018121491-What-is-a-Private-Window-with-Tor-Connectivity-) | ![w] ![l] ![m] | ✅ | It just works - Very slow ( less than 1 mbps; tested on ookla ) - Can just somehow stream 720p detailed motion videos - Captcha issues while Surfing the web; since TOR network so, obviously - Cloudflare bots protection checking leads to more delay in opening websites. But it works. |

Reviewer: ```suggestion
| [Tor via Brave Browser](https://support.brave.com/hc/en-us/articles/360018121491-What-is-a-Private-Window-with-Tor-Connectivity-) | ![w] ![l] ![m] | ✅ | It just works - Very slow ( less than 1 Mbps; tested on Ookla ) - Can just somehow stream 720p detailed motion videos - Captcha issues while Surfing the web; since TOR network so, obviously - Cloudflare bots protection checking leads to more delay in opening websites. But it works. |
```
